client/src/pages/Profile/ProfileDetail/ProfileDetailCard/ProfileDetailCard.tsx

```diff
@@ -0,0 +1,92 @@
+import Grid from '@mui/material/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import ImageList from '@mui/material/ImageList';
+import ImageListItem from '@mui/material/ImageListItem';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import Box from '@mui/material/Box';
+import useStyles from './useStyles';
+import { petImageInfo, Sitter } from '../sampleData';
+
+interface Props {
+  sitter: Sitter;
+  matches: boolean;
+}
+
+const ProfileDetailCard = ({ sitter, matches }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Grid item xs={12} sm={7} md={7} lg={6} xl={5} className={classes.root}>
+      <Box
+        width="100%"
+        height="100%"
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        className={classes.profileDetailBox}
+      >
+        <Paper elevation={6}>
+          <Box className={classes.profileDetailTop}>
+            <img
+              className={classes.profileBackgroundImg}
+              width="100%"
+              height="200px"
+              src={sitter.profileBackgroundImg}
+              alt="profile-detail-bg-image"
+            />
+            <Avatar
+              variant="circular"
+              className={classes.avatar}
+              src={sitter.userProfileImage}
+              alt={`${sitter.sitterFirstName}-${sitter.sitterLastName}-profile-image`}
+            />
+            <Typography component="h5" variant="h5" align="center" className={classes.sitterName}>
+              {`${sitter.sitterFirstName} ${sitter.sitterLastName}`}
+            </Typography>
+            <Typography component="h6" variant="h6" align="center" className={classes.sitterShortDesc}>
+              {sitter.sitterShortDesc}
+            </Typography>
+            <Box
+              width="250px"
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              className={classes.sitterLocation}
+            >
+              <LocationOnIcon className={classes.locationIcon} />
+              &nbsp;
```

Comment:
Using `&nbsp` entity directly is not a recommended best practice for react, change it if you can :-)

Reply:
Applied